### PR TITLE
add Link to A component

### PR DESCRIPTION
**Pull request by stephaniejnc**
issue #82 
- becomes a Link if color and weight are passed
- waiting on nwTheme.colors.primary to be changed

### CODE REVIEW

**Comment by stephaniejnc** [comment]
- note that for hackcampTheme, primary is the same as linkHover color in ThemeProvider
- for hacker app link, just pass color and weight into regular A component, otherwise pass nothing

nwTheme
hackcampTheme

File: src/pages/Charcuterie.js [hunk 1]
```diff
@@ -73,6 +73,15 @@ export default () => {
           contrast and complement each other's taste. It's some really long text. I'm really writing
           this way later than I should be. Is this what it's like to sell your soul to nwPlus?{' '}
         </P>
+        <br></br>
+        <A color={false} weight={false} href="https://nwplus.io">
```
Comment (panjenny0) [inline src/pages/Charcuterie.js:77]:
```suggestion
        <A href="https://nwplus.io">
```

File: src/components/Typography.js [hunk 2]
```diff
@@ -41,8 +41,9 @@ export const I = styled.i`
 export const A = styled.a`
   cursor: pointer;
   text-decoration: none;
-  border-bottom: 1px solid ${p => p.theme.colors.link};
-  color: ${p => p.theme.colors.link};
+  font-weight: ${p => (p.weight ? 600 : 400)};
```
Comment (panjenny0) [inline src/components/Typography.js:44]: I think this property should be named `bolded` to represent how it styles the text
```suggestion
  font-weight: ${p => (p.bolded ? 600 : 400)};
```

File: src/pages/Charcuterie.js [hunk 3]
```diff
@@ -73,6 +73,15 @@ export default () => {
...
+          This is a link.
+        </A>
+        <br></br>
+        <br></br>
+        <A color={true} weight={true} href="https://nwplus.io">
```
Comment (panjenny0) [inline src/pages/Charcuterie.js:82]:
```suggestion
        <A color="primary" bolded href="https://nwplus.io">
```

File: src/components/Typography.js [hunk 4]
```diff
@@ -41,8 +41,9 @@ export const I = styled.i`
...
+  border-bottom: 1px solid ${p => (p.color ? p.theme.colors.primary : p.theme.colors.link)};
+  color: ${p => (p.color ? p.theme.colors.primary : p.theme.colors.link)};
```
Comment (panjenny0) [inline src/components/Typography.js:46]: Just to make this component as reusable as possible, can we make it something like this?
```suggestion
  color: ${p => (p.color ? p.color === "primary" && p.theme.colors.primary || p.color : p.theme.colors.link)};
```

**Review by panjenny0 — CHANGES_REQUESTED** [review]
Good job! Left some comments to make the component as reusable as possible :)

**Comment by stephaniejnc** [comment]
thank you so much @panjenny0 :))

File: src/components/Typography.js [hunk 5]
```diff
@@ -41,8 +41,9 @@ export const I = styled.i`
 export const A = styled.a`
   cursor: pointer;
   text-decoration: none;
-  border-bottom: 1px solid ${p => p.theme.colors.link};
-  color: ${p => p.theme.colors.link};
+  font-weight: ${p => (p.bolded ? 600 : 400)};
+  border-bottom: 1px solid ${p => (p.color ? p.theme.colors.primary : p.theme.colors.link)};
+  color: ${p => (p.color ? p.color === "primary" && p.theme.colors.primary || p.color : p.theme.colors.link)};
```
Comment (panjenny0) [inline src/components/Typography.js:46]: lol I think my suggestion caused a linting error :angrycry:
can you add a bracket here?
```suggestion
  color: ${p => (p.color ? (p.color === "primary" && p.theme.colors.primary) || p.color : p.theme.colors.link)};
```

**Review by panjenny0 — APPROVED** [review]
good stuff!! :shipit:

**Pull request merged**